src/DataFixtures/ORM/LoadCommitteeMembershipHistoryData.php

```diff
@@ -67,6 +67,9 @@ private function createHistory(
 
     public function getDependencies()
     {
-        return [LoadAdherentData::class];
+        return [
+            LoadAdherentData::class,
```

Comment:
`LoadAdherentData` is no longer required here, because `LoadCommitteeData` depends on `LoadAdherentData`